frontend/packages/dev-console/src/components/monitoring/metrics/__tests__/MetricsQueryInput.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Button } from '@patternfly/react-core';
+import { Dropdown } from '@console/internal/components/utils';
+import { QueryInput } from '@console/internal/components/monitoring/metrics';
+import { MetricsQueryInput } from '../MetricsQueryInput';
+
+describe('Metrics Query Input', () => {
```

Comment:
Is it possible to add test cases to verify different states on drop down selection change?

Reply:
I tried that `simulate change`  is not work ing on dropdown component

Reply:
Updated the Specs, Thanks @christianvogt :)